src/diff/index.js

```diff
@@ -310,7 +304,7 @@ function diffElementNodes(
 ) {
 	let i;
 	let oldProps = oldVNode.props;
-	let newProps = newVNode.props;
+	let newProps = (newVNode.props = newVNode._node.props);
```

Comment:
Doesn't this skip the prop normalization that is done in createBackingNode?

Reply:
Yes still figuring it all out, it's pretty hard here since we don't have a sep `create` step